### ISSUE TITLE
Error in starting mysql, postgres, mongo etc etc after enabling volumes

### ISSUE THREAD

**Issue by sahin** [docker/kitematic#1326]
I am using latest version in Kitematic. After we are setting "Enable Volumes", we are getting the following the errors.

Postgres
````
The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.
The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".
Data page checksums are disabled.
fixing permissions on existing directory /var/lib/postgresql/data ... ok
creating subdirectories ... initdb: could not create directory "/var/lib/postgresql/data/global": Permission denied
initdb: removing contents of data directory "/var/lib/postgresql/data"
The files belonging to this database system will be owned by user "postgres".
````

Mysql
````
Initializing database
2015-12-28T01:24:37.592834Z 0 [Warning] Setting lower_case_table_names=2 because file system for /var/lib/mysql/ is case insensitive
2015-12-28T01:24:37.644847Z 0 [ERROR] InnoDB: Operating system error number 13 in a file operation.
2015-12-28T01:24:37.645642Z 0 [ERROR] InnoDB: The error means mysqld does not have the access rights to the directory.
2015-12-28T01:24:37.646569Z 0 [ERROR] InnoDB: Operating system error number 13 in a file operation.
2015-12-28T01:24:37.647002Z 0 [ERROR] InnoDB: The error means mysqld does not have the access rights to the directory.
2015-12-28T01:24:37.647438Z 0 [ERROR] InnoDB: Cannot open datafile './ibdata1'
2015-12-28T01:24:37.647820Z 0 [ERROR] InnoDB: Could not open or create the system tablespace. If you tried to add new data files to the system tablespace, and it failed here, you should now edit innodb_data_file_path in my.cnf back to what it was, and remove the new ibdata files InnoDB created in this failed attempt. InnoDB only wrote those files full of zeros, but did not yet use them in any way. But be careful: do not remove old data files which contain your precious data!
2015-12-28T01:24:37.648064Z 0 [ERROR] InnoDB: InnoDB Database creation was aborted with error Cannot open a file. You may need to delete the ibdata1 file before trying to start up again.
2015-12-28T01:24:38.252569Z 0 [ERROR] Plugin 'InnoDB' init function returned error.
2015-12-28T01:24:38.252688Z 0 [ERROR] Plugin 'InnoDB' registration as a STORAGE ENGINE failed.
2015-12-28T01:24:38.253262Z 0 [ERROR] Failed to initialize plugins.
2015-12-28T01:24:38.253551Z 0 [ERROR] Aborting
`````

Mongo

```
2015-12-28T01:26:54.678+0000 I CONTROL  [initandlisten] MongoDB starting : pid=1 port=27017 dbpath=/data/db 64-bit host=a484d3b2a3e0
2015-12-28T01:26:54.680+0000 I CONTROL  [initandlisten] db version v3.2.0
2015-12-28T01:26:54.682+0000 I CONTROL  [initandlisten] git version: 45d947729a0315accb6d4f15a6b06be6d9c19fe7
2015-12-28T01:26:54.683+0000 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.1e 11 Feb 2013
2015-12-28T01:26:54.683+0000 I CONTROL  [initandlisten] allocator: tcmalloc
2015-12-28T01:26:54.683+0000 I CONTROL  [initandlisten] modules: none
2015-12-28T01:26:54.683+0000 I CONTROL  [initandlisten] build environment:
2015-12-28T01:26:54.683+0000 I CONTROL  [initandlisten]     distmod: debian71
2015-12-28T01:26:54.683+0000 I CONTROL  [initandlisten]     distarch: x86_64
2015-12-28T01:26:54.683+0000 I CONTROL  [initandlisten]     target_arch: x86_64
2015-12-28T01:26:54.683+0000 I CONTROL  [initandlisten] options: {}
2015-12-28T01:26:54.702+0000 I STORAGE  [initandlisten] exception in initAndListen: 98 Unable to create/open lock file: /data/db/mongod.lock errno:13 Permission denied Is a mongod instance already running?, terminating
2015-12-28T01:26:54.702+0000 I CONTROL  [initandlisten] dbexit:  rc: 100
```

**Comment by thaJeztah** [comment]
I think this is a duplicate of https://github.com/docker/kitematic/issues/209

**Comment by sahin** [comment]
@thaJeztah closing this issue
